### PR TITLE
Add kolibri_explore_plugin

### PR DESCRIPTION
**Pull request by dylanmccall**
https://phabricator.endlessm.com/T33407

### CODE REVIEW

**Comment by manuq** [comment]
Is this ready for review? I see it is marked as draft.

**Comment by dylanmccall** [comment]
This is a draft pull request for the moment because we need some changes to kolibri-explore-plugin in order to support this work. In particular, we could use a way to locate the apps bundle //outside// of the Python package (the best would be something in options.ini so we can get it loading from `/android_asset/apps-bundle`), and we should try to reduce the size of this bundle as well, since it's going to bring us well over the play store's limit.

I'm working on those bits and will update here when those PRs are ready :) But hopefully this PR won't need to change all that much once the rest of it is in order.

File: src/initialization.py [hunk 1]
```diff
@@ -38,6 +38,8 @@
 
 os.environ["KOLIBRI_CHERRYPY_THREAD_POOL"] = "2"
 
+os.environ["KOLIBRI_APPS_BUNDLE_PATH"] = os.path.join(script_dir, "apps-bundle", "apps")
```
Comment (manuq) [inline src/initialization.py:41]: :+1:

File: Makefile [hunk 2]
```diff
@@ -38,7 +38,7 @@ needs-android-dirs:
 
 # Clear out apks
 clean:
-	- rm -rf dist/*.apk src/kolibri tmpenv
+	- rm -rf dist/*.apk src/kolibri src/apps-bundle tmpenv
```
Comment (manuq) [inline Makefile:41]: :+1:

File: Makefile [hunk 3]
```diff
@@ -62,10 +62,18 @@ get-whl: clean-whl
...
 	rm -r src/kolibri 2> /dev/null || true
-	unzip -qo "whl/kolibri*.whl" "kolibri/*" -x "kolibri/dist/py2only*" -d src/
+	unzip -qo "whl/kolibri-*.whl" "kolibri/*" -x "kolibri/dist/py2only*" -d src/
+	pip install --target=src --no-deps kolibri_explore_plugin
 	# patch Django to allow migrations to be pyc files, as p4a compiles and deletes the originals
 	sed -i 's/if name.endswith(".py"):/if name.endswith(".py") or name.endswith(".pyc"):/g' src/kolibri/dist/django/db/migrations/loader.py
 
+.PHONY: apps-bundle.zip
+apps-bundle.zip:
+	wget -N https://github.com/endlessm/kolibri-explore-plugin/releases/download/v2.0.15/apps-bundle.zip
```
Comment (manuq) [inline Makefile:72]: :+1:

File: Makefile [hunk 4]
```diff
@@ -62,10 +62,18 @@ get-whl: clean-whl
...
+
+src/apps-bundle: clean apps-bundle.zip
+	unzip -qo apps-bundle.zip -d src/apps-bundle
```
Comment (manuq) [inline Makefile:75]: :+1:

**Review by manuq — APPROVED** [review]

**Pull request merged**